lib/inspec/resource.rb

```diff
@@ -122,6 +122,7 @@ def self.validate_resource_dsl_version!(version)
 require 'resources/packages'
 require 'resources/parse_config'
 require 'resources/passwd'
+require 'resources/pg_hba_conf'
```

Comment:
Just like I mentioned in #1963, I'd like to suggest this resource be named `postgres_hba_conf` to keep all PostgreSQL-related resourced prefixed the same. While I know the file is named pg_hba.conf in practice, it's more important for the InSpec UX to be consistent.